src/components/fields/radio-field/README.md

```diff
@@ -0,0 +1,72 @@
+| `horizontalConstraint` | `object`           |          | `m`, `l`, `xl`, `scale` | `scale`          | Horizontal size limit of the input fields.                                                                                                                                                                                                                            |
+| `errors`               | `object`           |    -     | -                       | -                | A map of errors. Error messages for known errors are rendered automatically. Unknown errors will be forwarded to `renderError`.                                                                                                                                       |
+| `renderError`          | `function`         |    -     | -                       | -                | Called with custom errors, as `renderError(key, error)`. This function can return a message which will be wrapped in an `ErrorMessage`. It can also return `null` to show no error.                                                                                   |
+| `isRequired`           | `bool`             |    -     | -                       | `false`          | Indicates if the value is required. Shows an the "required asterisk" if so.                                                                                                                                                                                           |
+| `touched`              | `bool`             |    -     | -                       | `false`          | Indicates whether the field was touched. Errors will only be shown when the field was touched.                                                                                                                                                                        |
+| `name`                 | `string`           |    -     | -                       | -                | Used as HTML `name` of the input component. property                                                                                                                                                                                                                  |
+| `value`                | `string`           |    ✅    | -                       | -                | Value of the input component.                                                                                                                                                                                                                                         |
+| `onChange`             | `func`             |    -     | -                       | -                | Called with an event containing the new value. Required when input is not read only. Parent should pass it back as `value`.                                                                                                                                           |
```

Comment:
can a RadioGroup be read only?

Reply:
Maybe, maybe not. I doesn't hurt to have it I suppose, as it's inline with the other inputs/fields.